cpp/src/text/generate_ngrams.cu

```diff
@@ -109,11 +109,11 @@ std::unique_ptr<cudf::column> generate_ngrams(
                              if (d_strings.is_null(idx)) return false;
                              return !d_strings.element<cudf::string_view>(idx).empty();
                            },
-                           stream,
-                           mr)
+                           stream)
                            ->release();
     strings_count = table_offsets.front()->size() - 1;
-    return std::move(table_offsets.front());
+    auto result   = std::move(table_offsets.front());
+    return result;
```

Comment:
What is the difference from the previous version? Returning `std::move()` should has identical effect.

Reply:
I was just trying different methods to return the unique column ptr without using std::move in the return statement.
I felt this more cleanly communicated that attempt.

Reply:
Why is std::move required here? Why not `return table_offsets.front()`?

Reply:
It does not compile without the `std::move`

Reply:
`std::unique_ptr` is not copyable. You can return a `unique_ptr` from an `rvalue`, but not from a `lvalue` without `std::move` to convert it to an `rvalue`.